fastmath.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+import argparse
+import ast
+import importlib
+import pathlib
+
+
+def get_njit_funcs(pkg_dir):
+    """
+    Identify all njit functions
+
+    Parameters
+    ----------
+    pkg_dir : An instance of pathlib.Path
```

Comment:
@seanlaw 
Do you think the type of parameter `pkg_dir` should be `str`? Then, in the function, we can do: 
```
pkg_dir = pathlib.Path(pkg_dir)
```

----

On one hand, we run the script fastmath.py via terminal and that's all we need. On the other hand, I am thinking how easily one can use the function `get_njit_funcs`. Should I even bother to think about it? If yes, then I think passing path as `str` is easier than passing an instance of `pathlib.Path(...)`. 

What do you think?

Reply:
I agree. It should just be type `str` as it is the easiest to manage